imagekit/conf.py

```diff
@@ -17,25 +17,17 @@ class ImageKitConf(AppConf):
 
     def configure_cache_backend(self, value):
         if value is None:
-            try:
-                from django.core.cache.backends.dummy import DummyCache
-            except ImportError:
-                dummy_cache = 'dummy://'
-            else:
-                dummy_cache = 'django.core.cache.backends.dummy.DummyCache'
-
             # DEFAULT_CACHE_ALIAS doesn't exist in Django<=1.2
             try:
-                from django.core.cache import DEFAULT_CACHE_ALIAS as default_cache_alias
+                from django.core.cache import DEFAULT_CACHE_ALIAS
+                default_cache_alias = DEFAULT_CACHE_ALIAS or 'default'
```

Comment:
DEFAULT_CACHE_ALIAS is not a setting in Django. It is a constant from when it was introduced (in django 1.3) to nowadays. It was always `'default'` and probably it will be allays `'default'`. If the value changes for some reason probably the django cache framework will know what to do and we do not need to fallback to something.